web/services/getProjectUserWeeklyHoursReportJsonService.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/*
+ * Copyright (C) 2009 Igalia, S.L. <info@igalia.com>
```

Comment:
Same as in `GetProjectUserWeeklyHoursReportAction.php`, update the copyright year and indent with spaces instead of tabs.